CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 <!--next-version-placeholder-->
 
+## v0.21.2 (2021-10-05)
```

Comment:
No need to edit the version or changelog directly. We use semantic release so it will auto increment and fill out data.

Reply:
done